docs/src/pages/guides/FHIRValidationGuide.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: FHIR Validation Guide
+description: FHIR Validation Guide
+date:   2020-01-17 15:45:00 -0400
+permalink: /fhirvalidationguide/
+---
+
+## Overview
+
+The FHIR validation component (FHIRValidator) provides Java APIs for validating FHIR resources using constraints specified in their corresponding structure definitions. For example, in the Patient resource, we have the following constraint:
```

Comment:
```suggestion
The IBM FHIR Server Validation component (FHIRValidator) provides Java APIs for validating FHIR resources using constraints specified in their corresponding structure definitions. For example, in the Patient resource, we have the following constraint:
```

Reply:
do we want to add the path to the component?  on bintray or in github?

Reply:
added links to the components referenced